src/main/js/controls/Scatter/helpers/helpers.js

```diff
@@ -350,18 +350,15 @@ const drawDataPoints = (scale, config, pointGroupPath) => {
  *      * Shows/hides the regions
  *
  * @private
- * @param {object} graphContext - Graph instance
- * @param {Scatter} control - Scatter instance
  * @param {object} config - Graph config object derived from input JSON
  * @param {d3.selection} canvasSVG - d3 selection node of canvas svg
  * @returns {function()} callback function handler for RAF
  */
-const onAnimationHandler = (graphContext, control, config, canvasSVG) => () => {
-    control.redraw(graphContext);
+const onAnimationHandler = (config, canvasSVG) => () => {
     processRegions(config, canvasSVG);
 };
 /**
- * Click handler for legend item. Removes the data points from scatter graph when clicked and calls redraw
+ * Click handler for legend item. Removes the line from graph when clicked and calls redraw
```

Comment:
Um, now it doesn't call redraw, right?

Reply:
updated here https://github.com/cerner/carbon-graphs/pull/140/commits/e6eaf858e8e58510cc2e5b8f189e32df80b4059d.